src/commands/math.js

```diff
@@ -466,3 +466,23 @@ API.MathField = function(APIClasses) {
     };
   });
 };
+
+/**
+ * This is almost the same as StaticMath, except we don't
+ * bind any mouse events (which give us selectability and move
+ * around an invisible cursor).
+ *
+ * Otherwise, if you have a button that contains a rendered StaticMath symbol
+ * which causes something to be written into an editable mathquill,
+ * the mouseup event will hijack focus.
+ */
+API.InertMath = function(APIClasses) {
+  return P(APIClasses.StaticMath, function(_, super_) {
+    this.RootBlock = MathBlock;
+    _.__mathquillify = function() {
+      super_.__mathquillify.call(this, 'mq-math-mode');
+      this.__controller.staticMathTextareaEvents();
```

Comment:
as far as I know, `staticMathTextareaEvents()` is part of making mouse selection of StaticMath's work (that is, after mouse selecting, it lets you Cmd-C to copy the selection), and doesn't do anything else, so I don't think you even need this